src/org/opensolaris/opengrok/configuration/RuntimeEnvironment.java

```diff
@@ -1135,7 +1143,7 @@ public int getGroupsCollapseThreshold() {
      * @param file the file to read
      * @throws IOException if an error occurs
```

Comment:
Didn't make changes in the comment, as FileNotFoundException is IOException
https://docs.oracle.com/javase/7/docs/api/java/io/FileNotFoundException.html?is-external=true